packages/sparqlee/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@comunica/sparqlee",
+  "version": "2.8.2",
+  "description": "A simple SPARQL expression evaluator",
+  "main": "lib/index.js",
+  "typings": "lib/index",
+  "scripts": {
+    "build": "node \"../../node_modules/typescript/bin/tsc\"",
+    "bench": "node \"../../node_modules/typescript/bin/tsc\" && node benchmarks/*.js"
+  },
+  "files": [
+    "lib/**/*.d.ts",
+    "lib/**/*.js",
+    "lib/**/*.js.map",
+    "benchmarks/**/*.d.ts",
+    "benchmarks/**/*.js",
+    "benchmarks/**/*.js.map"
+  ],
+  "license": "MIT",
+  "keywords": [
+    "sparql",
+    "expression",
+    "rdf",
+    "evaluator"
+  ],
+  "author": "Wout Schellaert",
+  "devDependencies": {
+    "benchmark": "^2.1.4"
```

Comment:
Perhaps a limitation in your lerna.js workaround?

In any case, we could also just remove that specific linter rule for the benchmark dir, since it's non-production code anyways.